documentation/src/main/doc/modules/ROOT/pages/getting-started.adoc

```diff
@@ -79,15 +79,18 @@ Running the previous example should give the following output:
 
 [NOTE]
 ====
-SmallRye Reactive Messaging uses SLF4J as logger.
-Check the https://www.slf4j.org/manual.html[SLF4J documentation] to configure the logs.
+SmallRye Reactive Messaging uses JBoss-logger as logger.
+
 For testing purpose, you can simply add the following dependency to your `pom.xml` file:
 [source,xml]
 ----
 <dependency>
-  <groupId>org.slf4j</groupId>
-  <artifactId>slf4j-simple</artifactId>
-  <version>1.7.30</version>
+    <groupId>org.jboss.slf4j</groupId>
+    <artifactId>slf4j-jboss-logging</artifactId>
+    <version>${slf4j-jboss-logging.version}</version>
```

Comment:
This version should be set. The best is to use attributes for this. With Antora it's a bit messy:

* in documentation/src/main/antora/antora.yaml add a line for the JBoss logging version:
```
jboss-logging-version: '${jboss-logging.version}'
```

* in the documentation/pom.xml or in the root pom.xml declare the `jboss-logging.version` property

* in the getting-started document, use `{jboss-logging-version}` (no $) and [source,xml,subs=attributes+]